eng/pipelines/typespec-validation-all.yml

```diff
@@ -60,4 +60,3 @@ jobs:
       $(Build.SourcesDirectory)/eng/scripts/Validate-TypeSpec.ps1 -CheckAll -GitClean -Verbose
     displayName: Validate All Specs
     ignoreLASTEXITCODE: true
-    condition: succeededOrFailed()
```

Comment:
I have no idea why I added this condition in the first place.  Running `tsv` if `npm ci` failed just causes confusion, since the command can't even start.